key-rotator/manifest/manifest.go

```diff
@@ -63,7 +132,7 @@ type BatchSigningPublicKey struct {
 	// PKIX SubjectPublicKeyInfo structure. It must be an ECDSA P256 key.
 	PublicKey string `json:"public-key"`
 	// Expiration is the ISO 8601 encoded UTC date at which this key expires.
-	Expiration string `json:"expiration"`
+	Expiration string `json:"expiration,omitempty"`
```

Comment:
Careful with this -- it's possible that the MITRE portal server and the Apple/Google ingestion servers' manifest parsers require `expiration` to be present even if they ignore the value.

Reply:
Hmm, that's a good point.

Even if the other parties are reading this field, I strongly suspect no one is respecting it -- spot-checking a few prod-us manifests, many of these expiration dates have already passed. And IMO it's better for internal keys to not have a built-in expiration date -- when possible, rotation/versioning is a better solution than expiration -- so I think the ideal stable state would be for this field to not exist at all.

Do you have any suggestions here? I could write a date long in the future, but that feels hackish. Maybe we should either (a) ask [and risk no one having a clue what this field even is] or (b) skip writing it on a first pass and see who breaks.

Reply:
The field is absolutely being ignored, but I'm worried that someone's `json.Unmarshal` (or equivalent) call is going to fail because their parser marked the field as non-optional. I lean towards putting a far-future date in here--it's dumb, but we can be confident it won't break anyone and also it's arguably correct in the sense that maybe we will expire that key in the year 2525.